src/OSPSuite.R/MinimalImplementations/ProgressManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Utility.Events;
+
+namespace OSPSuite.R.MinimalImplementations
+{
+   public class ProgressManager : IProgressManager
```

Comment:
@Yuri05 Theoretically, that's where we could implement some progress stuff for R. Version 2 but possible